language/de-DE/com_content.ini

```
@@ -83,6 +83,7 @@ COM_CONTENT_RATINGS_COUNT="Bewertungen: %s"
 COM_CONTENT_READ_MORE="Weiterlesen &hellip; "
 COM_CONTENT_READ_MORE_TITLE="Weiterlesen: %s"
 COM_CONTENT_REGISTER_TO_READ_MORE="Anmelden, um den ganzen Beitrag zu lesen"
+COM_CONTENT_RUN_TRANSITION="Run Transition"
```

```suggestion
COM_CONTENT_RUN_TRANSITION="Übergang ausführen"
```
https://github.com/joomlagerman/joomla/blob/1f5d03b967c443a1e291a4bbf79205f106fa094a/administrator/language/de-DE/com_content.ini#L164